### PR TITLE
Using short hashes and minor tracing improvements for grpc copy

### PR DESCRIPTION
**Pull request by SergeyTeplyakov**

### CODE REVIEW

File: Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcContentServer.cs [hunk 1]
```diff
@@ -319,18 +311,18 @@ private async Task<(long Chunks, long Bytes)> StreamContentAsync(Stream reader,
...
 
                 // Read the next chunk while waiting for the response
-                await Task.WhenAll(ReadNextChunk(), responseStream.WriteAsync(response));
+                await Task.WhenAll(readNextChunk(), responseStream.WriteAsync(response));
             }
 
             return (chunks, bytes);
 
-            async Task<int> ReadNextChunk() { chunkSize = await reader.ReadAsync(buffer, 0, buffer.Length, ct); return chunkSize; }
+            async Task<int> readNextChunk() { chunkSize = await input.ReadAsync(buffer, 0, buffer.Length, ct); return chunkSize; }
```
Comment (JuanCarlosGI) [inline Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcContentServer.cs]: Is it convention to name local functions starting with lower case?

File: Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcContentServer.cs [hunk 2]
```diff
@@ -162,17 +167,14 @@ public async Task<GetStatsResponse> GetStatsAsync(GetStatsRequest request, Cance
 
         private async Task<OpenStreamResult> GetFileStreamAsync(Context context, ContentHash hash)
         {
-            Debug.Assert(_contentStoreByCacheName != null);
```
[the remark this answers is not in the document]
Comment (SergeyTeplyakov) [reply Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcContentServer.cs]: Yep, that was intentional. 

I think it make sense to have consistency in the codebase in terms of arguments validation.

File: Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcContentServer.cs (hunk 1 above)
Comment (SergeyTeplyakov) [reply Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcContentServer.cs]: Yep.

**Pull request merged**